FrostySdk/Managers/Loaders/DbObjectAssetLoader.cs

```diff
@@ -88,32 +90,67 @@ private bool LoadToc(string sbName, int superBundleId, DbObjectDict toc, ref Lis
         bool processBaseBundles = false;
         
         // process bundles
-        if (toc.ContainsKey("bundles"))
         {
```

Comment:
These braces are not needed

Reply:
okay this looks weird, i mean the braces surrounding the if (toc.ContainsKey("bundles")